### DISCUSSION TITLE
Retro Achievement support through rCheevos

### DISCUSSION THREAD

**Discussion by ZaneDragonBorn** [Swordfish90/Lemuroid#653] [Ideas]
Emulators such as RetroArch, melonds, and more feature support for Retro Achievements and it makes it really hard to pick this one over the rest due to this missing feature. It makes playing these games a whole lot more rewarding and I was hoping for support.

After digging for information, I've learned that people use [rcheevos](https://github.com/RetroAchievements/rcheevos) to add support. Though people did note that you have to implement the "network and UI layers" for it. Additionally [RAInterface](https://github.com/RetroAchievements/RAInterface/wiki/Creating-a-new-emulator) will outline things to consider when adding support. Though you can't use this as it requires RAIntegrate which is Windows only.

I may try my hand at adding support myself but I am fairly new to coding so currently gonna have find some time to break things on my end.

**Comment by Hozaai** [comment]
I've been really considering what's the most efficient way to play on, I sticked with retroarch for a while and figured it almost all out, things that it lacks at the low user friendly level it is, but what I'm missing now is multiplayer options that other standalone emulator easily have. Have you figured a way out? Or did you give up? If so, it's completely reasonable.

**Comment by Santabo** [comment]
Does anyone know if retroachievements will ever be added? I hope it does.
